src/main/java/taxi/controller/driver/GetMyCurrentCarsController.java

```diff
@@ -0,0 +1,29 @@
+package taxi.controller.driver;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Car;
+import taxi.service.CarService;
+
+@WebServlet(urlPatterns = "/drivers/cars")
```

Comment:
Use one approach for servlet mapping in all controllers classes for the code consistency.
1. `WebServlet` Annotation approach like in this controller class
or
2. add servlet mapping to the `web.xml`